src/sentry/api/serializers/models/organization_member/__init__.py

```diff
@@ -1,13 +1,16 @@
 from .base import OrganizationMemberSerializer
 from .expand.projects import OrganizationMemberWithProjectsSerializer
-from .expand.teams import OrganizationMemberWithTeamsSerializer
 from .response import OrganizationMemberSCIMSerializerResponse, SCIMMeta
 from .scim import OrganizationMemberSCIMSerializer
 
+from .expand.teams import OrganizationMemberWithTeamsSerializer  # isort:skip
+from .expand.roles import OrganizationMemberWithRolesSerializer  # isort:skip
```

Comment:
Why we we skip isort here?

Reply:
I'm having problems with circular imports otherwise:
- `/__init__.py`: 
```py
from .expand.roles import OrganizationMemberWithRolesSerializer
from .expand.teams import OrganizationMemberWithTeamsSerializer
```
- `/expand/roles.py`: 
```py
from .. import OrganizationMemberWithTeamsSerializer
```
Is there another way to fix this?

Reply:
Ahh fair enough. Other than refactoring things to avoid it there's no great alternative here.